build-tools/packages/build-cli/src/lib/sets.ts

```diff
@@ -0,0 +1,26 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+/**
+ * Returns the difference between two sets. That is, it returns a Set containing the items in Set A that are not in Set
+ * B.
+ *
+ * @param setA - A set.
+ * @param setB - A set.
+ * @returns A new set containing the difference of Set A and Set B.
+ *
+ * @remarks
+ *
+ * Implementation from {@link https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set}.
+ *
+ * @internal
+ */
+export function difference<T>(setA: Set<T>, setB: Set<T>) {
```

Comment:
Missing return type

Reply:
Also, should we add this to one of our utils packages? Wouldn't give you access right now, but this seems useful more generally.